src/System.Management.Automation/engine/Modules/ModuleCmdletBase.cs

```diff
@@ -686,7 +687,7 @@ internal PSModuleInfo LoadUsingMultiVersionModuleBase(string moduleBase, Manifes
                             parentModule,
                             moduleSpecification.Name,
                             rootedPath, // fileBaseName
-                            /*extension*/null,
+                                        /*extension*/null,
```

Comment:
this shouldn't be indented so far.  would be better to have the comment at the end like the others or even better change this to use named params

Reply:
Fixed.